task_manager.py

```diff
@@ -1,4 +1,28 @@
 class TaskManager:
 
-    def __init__(self):
-        self.tasks = {}  # {id,object}
+    def __init__(self): 
+        self.tasks = {
+            1: {"title": "Task 1", "description": "Description of task 1", "category": "Front-end", "completed": True},
```

Comment:
please remove this test data and it is not the right way to test it. you should create task objects and store it